base/docs/basedocs.jl

```diff
@@ -1800,4 +1800,13 @@ The syntax `a.b = c` calls `setproperty!(a, :b, c)`.
 """
 Base.setproperty!
 
+"""
+    eval(expr)
+
+Evaluate an expression in the current module. Every `Module` (except those defined
```

Comment:
Why not "Evaluate an expression in the global scope of the current module." as this is sometimes confusing folks using `eval` inside a function.